shapeshift/web
# feat: wire-up <Governance />

## Description

This wires up the `<Governance` component. This:
- wire-up the component as app code
- adds missing description for the component
- changes the skeleton to be only for the network-fetched data, but initially display the header and description
- adds CSP entries
- add env vars for boardroom network requests, and adds a `/` suffix to make it work with CSP
- removes useless consumption of `useTranslate` in `<CText />` and use `<Text />` with `translation` prop instead

Note: As per mockups, this is only present in the desktop version.

## Notice




## Pull Request Type


## Issue (if applicable)

closes #1828

## Risk

None that I can see, this is a new domain. CSPs are added and I've checked they work with the actual build, not just the dev server.

## Testing

- Run `yarn build && serve -s build`. This is needed to check that env vars actually work as they should, anything involving environment variables should be tested with the actual build.
- Go to the Fox Page
- Verify that Governance data is properly fetched with no CSP errors in the console

## Screenshots (if applicable)

### Skeleton

<img width="1290" alt="image" src="https://user-images.githubusercontent.com/17035424/172158432-4b24e9b7-7236-49e5-b116-c6be9bbda81c.png">


### One opportunity

<img width="1270" alt="image" src="https://user-images.githubusercontent.com/17035424/172157973-0bbaa200-3b7a-4cc6-ac4a-2516b54f57b8.png">

### Two opportunities

<img width="871" alt="image" src="https://user-images.githubusercontent.com/17035424/172158570-2ae8740d-efba-4585-9a2b-772efedddec5.png">
